### PR TITLE
extract keys, refactor, and clean up ml smoothing

### PR DESCRIPTION
**Pull request by harvey2phase**

### CODE REVIEW

File: neware_parser/management/commands/ml_smoothing.py [hunk 1]
```diff
@@ -908,6 +905,12 @@ def train_step(neighborhood, params, fit_args):
     )
 
 
+def get_loss(measured, predicted, mask, mask_2):
+    return tf.reduce_mean(
```
Comment (Samuel-Buteau) [inline neware_parser/management/commands/ml_smoothing.py:909]: this should be part of the incentive functions.

first, tf.square(x-y) is the same as incentive_inequality(x, Inequality.Equals, y, Level.Proportional)
second, if you had a function called incentive_masked(incentive, mask):
return tf.reduce_mean( incentive * mask)/tf.reduce_mean(1e-10 + tf.reduce_mean(mask))

then you could replace calls to get_loss by calls to incentive_inequality and incentive_masked.

(I think this is more reusable
